### PR TITLE
Fix #1341

### LINKED ISSUE #1341: After reload pirate is shown as english
**Describe the bug**
After I reloaded english is pirate until you change the language to anything else and back to english.

**To Reproduce**
1. Change the language to pirate
2. Reload

**Expected behavior**
English is pirate.

**Screenshots**
Screen Shot 2021-03-20 at 8 02 29

**What device are you on?**
MacBook Pro 16"

### PR DESCRIPTION
**Pull request by bansalarnav**
PR to fix #1341 

Apparently, i18n changes any characters after a `-` to capitals. So changing `en-pirate` to `en-PIRATE` and `sr-Latin` to `sr-LATIN` fixed the issue. I checked it and Latin also had the same issue where the selector showed Engilsh. Rest of the stuff changed is some formatting by prettier

### CODE REVIEW

File: kofta/src/app/components/LanguageSelector.tsx [hunk 1]
```diff
@@ -67,11 +67,12 @@ export const LanguageSelector: React.FC<LanguageSelectorProps> = ({
     { value: "gsw", label: "🇨🇭 Schwiizerdütsch" }, // Swiss German
 
     /* Other languages */
-    { value: "en-pirate", label: "🏴‍☠️ Pirate" },
+    { value: "en-PIRATE", label: "🏴‍☠️ Pirate" },
```
Comment (amitojsingh366) [inline kofta/src/app/components/LanguageSelector.tsx:70]: Did your run this locally and check if it fixes the issue?

Comment (amitojsingh366) [reply kofta/src/app/components/LanguageSelector.tsx:70]: just making sure

Comment (bansalarnav) [reply kofta/src/app/components/LanguageSelector.tsx:70]: > Did your run this locally and check if it fixes the issue?

Yep. I made the PR after I was sure that it fixes the issue and doesn't break anything else. All translations are working fine too

Comment (amitojsingh366) [reply kofta/src/app/components/LanguageSelector.tsx:70]: Sure then it seems fine to me!

**Review by amitojsingh366 — APPROVED** [review]

File: kofta/src/app/components/LanguageSelector.tsx [hunk 2]
```diff
@@ -67,11 +67,12 @@ export const LanguageSelector: React.FC<LanguageSelectorProps> = ({
...
     { value: "owo", label: "OwO Engwish" },
   ],
 }) => {
   const { i18n } = useTranslation();
+  console.log(i18n)
```
Comment (goldyydev) [inline kofta/src/app/components/LanguageSelector.tsx:75]:
```suggestion
```

**Review by goldyydev — APPROVED** [review]

**Pull request merged**